bin/varnishd/cache/cache_filter.h

```diff
@@ -58,7 +58,7 @@ struct vfp {
 	vfp_init_f		*init;
 	vfp_pull_f		*pull;
 	vfp_fini_f		*fini;
-	const void		*priv1;
+	const void		*vfp_priv;
```

Comment:
Should we go further and rename `vfp_entry`'s `priv1` to `body_priv` and `priv2` to something also more descriptive?

Reply:
yes, but that might bring even more sensible changes, so I would like to keep it out of here.